cpp-package/example/inference/inception_inference.cpp

```diff
@@ -62,6 +63,13 @@ class Predictor {
  private:
     void LoadModel(const std::string& model_json_file);
     void LoadParameters(const std::string& model_parameters_file);
+    void SplitParamMap(const std::map<std::string, NDArray>& paramMap,
+        std::map<std::string, NDArray>& argParamInTargetContext,  // NOLINT(*)
+        std::map<std::string, NDArray>& auxParamInTargetContext,  // NOLINT(*)
+        Context targetContext);
+    void ConvertParamMapToTargetContext(const std::map<std::string, NDArray>& paramMap,
+        std::map<std::string, NDArray>& paramMapInTargetContext, // NOLINT(*)
+        Context targetContext);
```

Comment:
Any reason you used non-const references instead of pointers against style recommendation ?

Reply:
sure, will pass in a pointer instead.